code/src/index.css

```diff
@@ -5,9 +5,169 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: url(./images/map.jpg);
+  font-size: 20px;
 }
 
+
+
+ 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.title {
+  position: absolute;
+  left: 20%;
+  right: 20%;
+  top: 10%;
+  font-family: 'Cinzel', serif;
+  background-color: antiquewhite;
+  width: fit-content;
+}
+
+.question {
+ position: absolute;
+ top: 30%;
+ left: 20%;
+ background-color: antiquewhite;
+ font-size: 27px;
+ display: flex;
+ text-align: left
+}
+
+.book-selection {
+position: absolute;
+ top: 40%;
+ left: 20%;
+ background-color: antiquewhite;
+ display: flex;
+ flex-direction: column;
+}
+
+.language-selection {
+  position: absolute;
+ top: 40%;
+ left: 20%;
+ background-color: antiquewhite;
+ font-size: 16px;
+    font-family: 'Cinzel', serif;
+    cursor: pointer;
+}
+
+.version-selection {
+  position: absolute;
+   top: 40%;
+   left: 20%;
+   background-color: antiquewhite;
+   display: flex;
+   flex-direction: column;
+  }
+
+  .results {
+    position: absolute;
+   top: 40%;
+   left: 20%;
+   background-color: antiquewhite;
+   display: flex;
+   flex-direction: column;
+  }
+
+.next-step {
+position: absolute;
+top: 60%;
+left: 20%;
+}
+
+.next-step button {
+    border-radius: 4px;
+    background-color: antiquewhite;
+    border: none;
+    text-align: center;
+    font-size: 32px;
+    padding: 16px;
+    width: 220px;
+    transition: all 0.5s;
+    cursor: pointer;
+    margin: 36px;
+    box-shadow: 0 10px 20px -8px rgba(0, 0, 0,.7);
+  }
+  
+  button{
+    cursor: pointer;
+    display: inline-block;
+    position: relative;
+    transition: 0.5s;
+  }
+  
+  button:after {
+    content: '»';
+    position: absolute;
+    opacity: 0;  
+    top: 14px;
+    right: -20px;
+    transition: 0.5s;
+  }
+  
+  button:hover{
+    padding-right: 24px;
+    padding-left:8px;
+  }
+  
+  button:hover:after {
+    opacity: 1;
+    right: 10px;
+  }
+
+  .email {
+   width: 60%
+  } 
+
+
+  input[type="email"]::placeholder {   
+    font-size: 16px;
+    font-family: 'Cinzel', serif;
+    }
+
+    .submit {
+      font-family: 'Cinzel', serif;
+        border-radius: 4px;
+        background-color: antiquewhite;
+        border: none;
+        text-align: center;
+        font-size: 16px;
+        padding: 10px;
+        width: 100px;
+        transition: all 0.5s;
+        cursor: pointer;
+        margin: 20px;
+        box-shadow: 0 10px 20px -8px rgba(0, 0, 0,.7);
+      }
+      
+      .submit:after {
+        position: absolute;
+        opacity: 0;  
+        top: 14px;
+        right: -20px;
+        transition: 0.5s;
+      }
+      
+      .submit:hover{
+        padding-right: 24px;
+        padding-left:8px;
+        font-size: 20px;
+      }
+      
+      .submit:hover:after {
+        opacity: 1;
+        right: 10px;
+      }
+    
+    
+      @media (min-width: 1024px) {
```

Comment:
I hope you have the time to make it responsive! (now  it's a bit hard to see in mobile view) I don't think it was a must for this week though 👍  I understand that the map would be pretty hard to fit though 😄